apps/ledger-live-desktop/src/renderer/screens/platform/LiveApp.tsx

```diff
@@ -60,14 +60,15 @@ export function LiveApp({ match, appId: propsAppId, location }: Props) {
   }, [search, customDappUrl, urlParams?.customDappUrl, internalParams?.customDappUrl]);
 
   const handleClose = useCallback(() => {
-    if (returnTo.startsWith("/swap")) {
+    if (typeof returnTo === "string" && returnTo.startsWith("/swap")) {
```

Comment:
LGTM like that, another option could have been to simply do that
```suggestion
    if (returnTo?.startsWith("/swap")) {
```
Also if you update the type in the memo above you can get typescript to show the undefined correctly as shown in the screenshot below 
<img width="508" alt="Screenshot 2024-11-28 at 09 21 11" src="https://github.com/user-attachments/assets/93592874-a5fa-43f2-8d39-6ffc44fe4f32">

Reply:
Just pushed this change.
The video in the description is still valid after this commit.